### PR TITLE
Change version of modules and setup slides folder

### PR DESCRIPTION
**Pull request by Nirmay15**

### CODE REVIEW

File: packages/sheets/src/ui/components/search-bar/search-bar.tsx [hunk 1]
```diff
@@ -46,6 +46,12 @@ export const TSSearchBar = () => {
           '.data-panel-module__headerToggleBtn': {
             display: 'none !important',
           },
+          '.accordion-module__accordionItem:nth-child(2)': {
+            display: 'none !important',
+          },
+          '.accordion-module__accordionItem:nth-child(3)': {
+            display: 'none !important',
+          },
```
Comment (ashubham) [inline packages/sheets/src/ui/components/search-bar/search-bar.tsx:54]: wasnt this already done ?

File: packages/slides/src/ui/components/home/home.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+export const Home = () => {
+  const loader = useLoader();
+  const { run } = useShellContext();
+
+  return (
+    <Vertical className="home">
+      <Button
+        onClick={() => console.log('Clicks clicked')}
+        text={'Browse Content'}
```
Comment (ashubham) [inline packages/slides/src/ui/components/home/home.tsx:16]: I18n

File: packages/sheets/src/ui/components/search-bar/search-bar.tsx (hunk 1 above)
Comment (Nirmay15) [reply packages/sheets/src/ui/components/search-bar/search-bar.tsx]: Yes it is already in, removed them.

File: packages/slides/src/ui/components/home/home.tsx (hunk 2 above)
Comment (Nirmay15) [reply packages/slides/src/ui/components/home/home.tsx]: This was just for the setup, added browse thoughtspot to string json, will confirm exact strings for refresh buttons from nico on monday.

**Review by ashubham — APPROVED** [review]

**Pull request merged**